examples/angular/src/tsconfig.json

```diff
@@ -25,13 +25,16 @@
 
             // Add all possible bazel-out directories
             // See https://github.com/microsoft/TypeScript/issues/37257
+            "../bazel-out/darwin_arm64-fastbuild/bin/src",
             "../bazel-out/darwin-dbg/bin/src",
             "../bazel-out/darwin-fastbuild/bin/src",
-            "../bazel-out/darwin_arm64-fastbuild/bin/src",
+            "../bazel-out/darwin-opt-exec-2B5CBBC6/bin/src",
```

Comment:
do you know what the `2B5CBBC6` is based on? I hope it's a stable checksum that will match on another machine.

Reply:
John Cater says "the hex tag is just a hash of the execution platform" (https://github.com/bazelbuild/bazel/issues/12403#issuecomment-732952252) so it should be stable. Guessing it is a hash of "host" if the exec platform is the host since it is stable between k8, darwin & windows in these paths